specs/agents/tracing-instrumentation-messaging.md

```diff
@@ -188,6 +197,25 @@ queues/topics/exchanges will be ignored.
 | Central config | `true` |
 
 
+### Message metadata
+
+To support distributed tracing with automatic instrumentation, the messaging
+system must provide a mechanism to add metadata/properties/attributes to
+individual messages, akin to HTTP headers. If an APM agent supports
+trace-context for a given messaging system, it MUST use the following mechanisms
+so that cross-language tracing works:
+
+| Messaging system       | Mechanism |
+| ---------------------- | --------- |
+| Azure Queue            | Unknown |
```

Comment:
IIUC, Azure Queue Messages do not have a means to transport metadata / headers / attributes.